pkg/eventcache/eventcache.go

```diff
@@ -99,20 +98,25 @@ func (ec *Cache) handleEvents() {
 	tmp := ec.cache[:0]
 	for _, event := range ec.cache {
 		var err error
+		var callFn uint32
 		if getNsPid, ok := event.msg.(interface{ GetNsPid() uint32 }); ok {
 			nspid := getNsPid.GetNsPid()
 			err = handleExecEvent(&event, nspid)
+			callFn = 1
 		} else {
 			err = handleEvent(&event)
+			callFn = 2
 		}
 
 		if err != nil {
+			logger.GetLogger().WithError(err).Warnf("handleEvents() - %d - %d", callFn, event.color)
```

Comment:
I think this is going to spam our logs with a lot of messages, since we expect err to not be nil quite frequently. I would suggest moving it to Debugf

Reply:
Just trying to debug the issue here. I don't plan to keep them when the issue is resolved. (Same for the other log messages)